### PR TITLE
Updated apiDefinition.swagger.json

### PR DESCRIPTION
**Pull request by gulshankhurana**
- Changed the name from 'FCA' to 'Financial Conduct Authority UK' and also updated the description.
- Fixed the 'Common Search' action.
- Added 'Find Firm Details' and 'Find Product Details' actions.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Con-Def

Con-Test-CommonSearch-Individual

Con-Test-CommonSearch-Fund

Con-Test-CommonSearch-Firm

Con-Test-FirmDetails

Con-Test-ProductDetails

Flow-CommonSearch-Individual

Flow-CommonSearch-Firm

Flow-CommonSearch-Fund

Flow-FindFirmDetailsByFRN

Flow-FindProductDetailsByPRN

### CODE REVIEW

**Comment by gulshankhurana** [comment]
@vmanoharas, I have updated the apiDefinition.swagger.json as per your advice under #3242

File: independent-publisher-connectors/Financial Conduct Authority (FCA) UK Connector/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "FCA",
-    "description": "The online Financial Services Register is a public record of financial services firms, individuals and other bodies regulated by the Financial Conduct Authority (FCA)",
+    "title": "Financial Conduct Authority (FCA) UK",
```
Comment (vmanoharas) [inline independent-publisher-connectors/Financial Conduct Authority (FCA) UK Connector/apiDefinition.swagger.json]: The 'title' value goes over the character limit '30' ValuePath: /info/title. Kindly fix this.

**Review by vmanoharas — COMMENTED** [review]
Hello @gulshankhurana, 

Kindly fix the review comments.

File: independent-publisher-connectors/Financial Conduct Authority (FCA) UK Connector/apiDefinition.swagger.json (hunk 1 above)
Comment (gulshankhurana) [reply independent-publisher-connectors/Financial Conduct Authority (FCA) UK Connector/apiDefinition.swagger.json]: @vmanoharas , Fixed as advised. Thanks.

**Comment by gulshankhurana** [comment]
@vmanoharas , fixed the issue as advised. Thanks.

**Review by vmanoharas — DISMISSED** [review]
Hello @gulshankhurana 
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by gulshankhurana** [comment]
Hi @vmanoharas
I have fixed action errors caused by change in api definition by the api provider. I also added 2 more actions.
Please approve at your earliest convenience as some people are eagerly awaiting these changes to take effect so that they can start using this connector asap.

Kind regards
Gulshan

**Review by vmanoharas — APPROVED** [review]
Hello @gulshankhurana 
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by vmanoharas** [comment]
Hello @gulshankhurana, 

This PR is failing to upload into our validation tools, kindly close this PR and create new fresh PR.

**Comment by gulshankhurana** [comment]
Hi @vmanoharas 

As suggested, I have opened a new PR #3495 . 

Thanks.
Gulshan

**Pull request closed without merging**